EDEngineer/Resources/Data/localization.json

```diff
@@ -1718,12 +1718,6 @@
       "de": null,
       "fr": null,
       "ru": null
-    },

```

Comment:
Hi. Sorry, I didn´t mean to do that. I dont know this github system, I am just used to Tortoise/subversion.. Do I have to fix it still?